### PR TITLE
campaign component added to nav bar

### PR DESCRIPTION
**Pull request by GeekG1rl**

### CODE REVIEW

**Comment by GeekG1rl** [comment]
Don't merge yet 🍍

File: src/components/Campaigns/CampaignCreate.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Row, Col } from 'react-bootstrap';
+import { push } from 'react-router-redux';
+import tv4 from 'tv4';
```
Comment (venil7) [inline src/components/Campaigns/CampaignCreate.js]: prefer to use `Validator` from `componets/Validator`

File: src/components/Campaigns/CampaignDetails.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    const valid = validator.validate(campaign, schema);
+    if (valid) {
+      const { params: { id } } = this.props;
+      await dispatch(campaignUpdateThunk(id, campaign));
+      return dispatch(initialize(FORM_NAME, campaign, fields));
+    }
+    return dispatch(campaignValidationError(validator.error));
+  }
+
+  async handleCampaignDelete() {
```
Comment (venil7) [inline src/components/Campaigns/CampaignDetails.js]: this function is not called anywhere,
I'd suggest we just mark them as `Inactive` instead

File: src/components/Campaigns/CampaignDetails.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+// import CampaignActions from './CampaignActions';
+// import CampaignConfirmDeleteDialog from './CampaignConfirmDeleteDialog';
+import { push } from 'react-router-redux';
+import { dialogs } from 'decorators';
+import config from 'config';
+import validator from 'components/Validator/validator';
+
+const FORM_NAME = 'campaignForm';
+
+@dialogs()
```
Comment (venil7) [inline src/components/Campaigns/CampaignDetails.js]: You probably dont need anything todo with dialogs in this file

**Pull request merged**